packages/ember/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "target": "es2017",
+    "target": "es2020",
```

Comment:
Why did we bump this?

Reply:
I would prefer not to bump the target if we don't need to.

Reply:
sure thing, I reverted it!